<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFaultGradientSetting</class>
 <widget class="QDialog" name="CFaultGradientSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>开出量设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSttTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="m_tabBIn">
        <attribute name="title">
         <string>开入量</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="2">
            <widget class="QSttCheckBox" name="m_cbBinC">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QSttCheckBox" name="m_cbBinA">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSttCheckBox" name="m_cbBinB">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSttCheckBox" name="m_cbBinD">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QSttCheckBox" name="m_cbBinE">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>E</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QSttCheckBox" name="m_cbBinF">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>F</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSttCheckBox" name="m_cbBinG">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>G</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QSttCheckBox" name="m_cbBinI">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSttCheckBox" name="m_cbBinH">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>H</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QSttCheckBox" name="m_cbBinJ">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>J</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="m_rbAnd">
             <property name="text">
              <string>逻辑与</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="m_rbOr">
             <property name="text">
              <string>逻辑或</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="m_tabBOut">
        <attribute name="title">
         <string>开出量</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="m_gbNormal">
           <property name="title">
            <string>常态</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="2">
               <widget class="QSttCheckBox" name="m_cbBOut2">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>3</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpNormal</string>
                </attribute>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QSttCheckBox" name="m_cbBOut3">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>4</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpNormal</string>
                </attribute>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QSttCheckBox" name="m_cbBOut5">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>6</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpNormal</string>
                </attribute>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSttCheckBox" name="m_cbBOut6">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>7</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpNormal</string>
                </attribute>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSttCheckBox" name="m_cbBOut1">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>2</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpNormal</string>
                </attribute>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QSttCheckBox" name="m_cbBOut9">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>10</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpNormal</string>
                </attribute>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSttCheckBox" name="m_cbBOut7">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>8</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpNormal</string>
                </attribute>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QSttCheckBox" name="m_cbBOut4">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>5</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpNormal</string>
                </attribute>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QSttCheckBox" name="m_cbBOut8">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>9</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpNormal</string>
                </attribute>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QSttCheckBox" name="m_cbBOut0">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>1</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpNormal</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="m_gbError">
           <property name="title">
            <string>故障态</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="4">
               <widget class="QSttCheckBox" name="m_cbBOut10_E19">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>10</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpError</string>
                </attribute>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QSttCheckBox" name="m_cbBOut4_E13">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>4</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpError</string>
                </attribute>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSttCheckBox" name="m_cbBOut2_E11">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>2</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpError</string>
                </attribute>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSttCheckBox" name="m_cbBOut7_E16">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>7</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpError</string>
                </attribute>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QSttCheckBox" name="m_cbBOut5_E14">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>5</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpError</string>
                </attribute>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QSttCheckBox" name="m_cbBOut6_E15">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>6</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpError</string>
                </attribute>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QSttCheckBox" name="m_cbBOut9_E18">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>9</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpError</string>
                </attribute>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QSttCheckBox" name="m_cbBOut1_E10">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>1</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpError</string>
                </attribute>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSttCheckBox" name="m_cbBOut8_E17">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>8</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpError</string>
                </attribute>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QSttCheckBox" name="m_cbBOut3_E12">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>3</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">btnGrpError</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnOK">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnCancel">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSttTabWidget</class>
   <extends>QTabWidget</extends>
   <header>../Controls/SttTabWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QSttCheckBox</class>
   <extends>QCheckBox</extends>
   <header>../Controls/SttCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btnGrpNormal">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="btnGrpError">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
